src/main/java/com/powsybl/network/conversion/server/NetworkConversionService.java

```diff
@@ -150,24 +147,39 @@ void setGeoDataServerRest(RestTemplate geoDataServerRest) {
     }
 
     public ExportNetworkInfos exportCgmesSv(UUID networkUuid, List<UUID> otherNetworksUuid) throws XMLStreamException {
-        Network network = networksListToMergedNetwork(networkUuid, otherNetworksUuid);
+        Network mergedNetwork = networksListToMergedNetwork(networkUuid, otherNetworksUuid);
 
         Properties properties = new Properties();
         properties.put("iidm.import.cgmes.profile-used-for-initial-state-values", "SV");
 
         ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
         XMLStreamWriter writer = null;
+
+        List<Network> networks = new ArrayList<>();
+        networks.add(getNetwork(networkUuid));
+        otherNetworksUuid.forEach(uuid -> networks.add(getNetwork(uuid)));
```

Comment:
The networks are already created in networksListToMergedNetwork method (with call to getNetwork method)
So why calling again getNetwork here ?

Reply:
Yes that wasn't optimized. I fixed it